tests/conftest.py

```diff
@@ -103,6 +113,11 @@ async def async_http_request(
         {"sid": "uuid:dummy-avt1", "timeout": "Second-150"},
         "",
     ),
+    ("SUBSCRIBE", "http://dlna_dmr:1234/upnp/event/QPlay"): (
+        200,
+        {"sid": "uuid:dummy-avt1", "timeout": "Second-150"},
```

Comment:
The `sid` should be different to the others (so it doesn't clash in subscribe calls). Maybe `uuid:dummy-qp1`.